.travis/release.properties

```diff
@@ -30,7 +30,7 @@
 # Property used to set version of the release build.
 # To perform release build this property should be set to expected version
 # and the file has to be committed with the message "Release ${releaseVersion}"
-releaseVersion=0.5
+releaseVersion=0.6
```

Comment:
Could you please sync with SabreOSS:master?

Reply:
Done.